config/samples/ocm-api-secret.yaml

```diff
@@ -2,6 +2,7 @@
 
 apiVersion: v1
 data:
```

Comment:
A sample that doesn't show the data base64 encoded should use `stringData` like this example: https://kubernetes.io/docs/concepts/configuration/secret/#basic-authentication-secret

Reply:
Updated the field to use `stringData`.